package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "mocha && xo --extend probot"
+    "test": "mocha && standard --extend probot"
```

Comment:
nit: `--extend probot` probot is no longer needed, it should be `mocha && standard`